X11-xft.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.2
 name:          X11-xft
-version:       0.3.3
+version:       0.3.1
```

Comment:
You'd need to revert https://github.com/xmonad/X11-xft/commit/0f0066a71eef5ac8d879c84a42f0622a38cc12fa for this to make sense